resources/tests/msr/msr_reader.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Helper script used to read MSR values from ranges known to contain MSRs.
+
+print_msr() {
+    local msr_hex=$(printf "%#x" $1)
+    if output=$(rdmsr $msr_hex 2>> /dev/null); then
+        echo "$msr_hex,implemented,$output"
+    else
+        echo "$msr_hex,unimplemented,NaN"
+    fi
+}
+
+echo "MSR_ADDR,STATUS,VALUE"
```

Comment:
Any particular purpose for the `Status` field? I couldn't find a usage for it in the test.

Reply:
Yes, it's either `implemented` or `unimplemented`. It is used in `test_cpu_rdmsr`.